skrub/tests/test_table_vectorizer.py

```diff
@@ -642,6 +643,120 @@ def test__infer_date_format() -> None:
     assert _infer_date_format(date_column) is None
 
 
+@pytest.mark.parametrize(
+    "specific_transformers",
+    [
+        (MinHashEncoder(), ["str1", "str2"]),
+        (StandardScaler(), ["float"]),
+        ("mh_cat1", MinHashEncoder(), ["cat1"]),
+    ],
+)
+def test_specifying_specific_column_transformer(specific_transformers) -> None:
+    X = _get_dirty_dataframe()
+
+    tv = TableVectorizer(
+        specific_transformers=[specific_transformers],
+    )
+    X_enc_tv = tv.fit_transform(X)
+
+    # For the following section, we will use a different syntax using the
+    # column transformer to assert that the internals of the TableVectorizer
+    # work as expected.
+    # To do that, we get the assignment done by the default TableVectorizer
+    # (no arguments passed), and extend it with the column specific transformer
+    # which we then pass to the ColumnTransformer.
+    # We fit_transform the ColumnTransformer, and expect the same
+    # transformation (albeit, not necessarily in the same order) as the
+    # TableVectorizer.
+    if len(specific_transformers) == 2:
```

Comment:
That seems a bit too complex / copy-pasted from the code of the TableVectorizer. Could we just hard-code a couple of solutions, it would be easier to read for people in the long term

Reply:
This still needs addressing

Reply:
Fixed, thanks for the reminder!